docs/contributing/index.md

```diff
@@ -43,6 +43,28 @@ Once you've completed these steps, you are ready to submit your contribution.
 
 Once you've completed these steps, you are ready to submit your contribution.
 
+
+### Contributing to Documentation
+
+#### Documentation Style
+
+To ensure that our code is well-documented and easy to understand, we use [Google-style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) to document all functions, classes, and methods in this library.
+
+#### Getting Help with Setup and Configuration
+
+To work on documentation locally, follow these steps:
+
+- To setup the documentation, navigate to the scripts directory
+- Run the `scripts/build-docs.sh` script to build the documentation
+- Serve the documentation locally, using the `scripts/docs-live.sh` script
+
+If you encounter any issues while setting up the documentation using the provided steps, please refer to our [tutorial]( https://www.youtube.com/watch?v=mNjlMZ4F3So) for additional guidance.
+
+#### Caveats and Considerations
```

Comment:
```suggestion
##### Caveats and considerations
```

Indented one extra level so that it's nested underneath the local development section